jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -730,215 +730,6 @@ properties:
             type: number
             fits_keyword: PA_APER
             fits_hdu: SCI
```

Comment:
For now just move the individual keyword from the block. That means there won't be a block header/comment for it in the SCI header, but that's OK.